PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PayLaterButton.kt

```diff
@@ -72,6 +75,14 @@ class PayLaterButton @JvmOverloads constructor(
         )
     }
 
+    private fun updateColorFrom(typedArray: TypedArray) {
```

Comment:
Not to expand the scope of this PR, but does this same bug exist for the `PayPalCreditButton` and `PayPalButton`?

Reply:
No, those didn't have changes.
The reason this broke is because of the change in inheritance.